src/cpp/session/CMakeLists.txt

```diff
@@ -104,7 +104,7 @@ if(WIN32)
 endif()
 
 # include files
-file(GLOB_RECURSE SESSION_HEADER_FILES "*.h*")
+file(GLOB_RECURSE SESSION_HEADER_FILES CONFIGURE_DEPENDS "*.h*")
```

Comment:
What's this for?

Reply:
From https://cmake.org/cmake/help/latest/command/file.html#filesystem:

> New in version 3.12: If the CONFIGURE_DEPENDS flag is specified, CMake will add logic to the main build system check target to rerun the flagged GLOB commands at build time. If any of the outputs change, CMake will regenerate the build system.

In other words, it lets you add new header files, and CMake will automatically detect those new files on build without needing to re-run the CMake generator. Essentially, a small quality-of-life developer change.